Floofbot/Modules/Administration.cs

```diff
@@ -293,9 +293,17 @@ public async Task RemoveAutoBan([Summary("user ID")] string userId)
             }
 
             IUser badUser = resolveUser(user);
+            ulong uid = 0; // used if no resolved user
             if (badUser == null) {
-                await Context.Channel.SendMessageAsync($"⚠️ Could not find user \"{user}\"");
-                return;
+                if (Regex.IsMatch(user, @"\d{17,18}"))
```

Comment:
While it's very unlikely, it should be possible to have a user ID that's less than 17 characters long (and in the near future, more than 18). To be on the safe side, I'd run with `\d{16,}`. I see that this regex pattern is also used in the `resolveUser` helper function, so that could be changed as well.

This also applies to R369

(Side note, I think the `resolveUser` logic for a name in the format of `<name>#<discrim>` could have a bug. If a user is named `foo#bar#1234`, the logic after the passed regex check will pass in the arguments `foo` and `bar`, not `foo#bar` and `1234`. May need to resort to substrings to fix that one.)